Description: Upgrade packages for animal-sniffer to support
 latest 1.7 revision.
Author: James Page <james.page@ubuntu.com>
Forwarded: https://github.com/jenkinsci/jenkins/commit/dd8cdda6eeb4a29d991a61d58e06167b51ffa20f

Index: jenkins/core/src/main/java/hudson/Functions.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/Functions.java	2012-02-10 15:39:22.669867336 +0000
+++ jenkins/core/src/main/java/hudson/Functions.java	2012-02-28 16:49:18.501824775 +0000
@@ -63,7 +63,7 @@
 import org.apache.commons.jelly.XMLOutput;
 import org.apache.commons.jexl.parser.ASTSizeFunction;
 import org.apache.commons.jexl.util.Introspector;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 import org.jvnet.tiger_types.Types;
 import org.kohsuke.stapler.Ancestor;
 import org.kohsuke.stapler.Stapler;
Index: jenkins/core/src/main/java/hudson/Util.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/Util.java	2012-02-10 15:39:22.673867336 +0000
+++ jenkins/core/src/main/java/hudson/Util.java	2012-02-28 16:49:18.505824788 +0000
@@ -42,7 +42,7 @@
 import org.jruby.ext.posix.FileStat;
 import org.jruby.ext.posix.POSIX;
 import org.kohsuke.stapler.Stapler;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 
 import javax.crypto.SecretKey;
 import javax.crypto.spec.SecretKeySpec;
Index: jenkins/core/src/main/java/hudson/diagnosis/HudsonHomeDiskUsageChecker.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/diagnosis/HudsonHomeDiskUsageChecker.java	2012-02-10 15:39:22.677867336 +0000
+++ jenkins/core/src/main/java/hudson/diagnosis/HudsonHomeDiskUsageChecker.java	2012-02-28 16:49:18.505824788 +0000
@@ -26,7 +26,7 @@
 import hudson.Extension;
 import jenkins.model.Jenkins;
 import hudson.model.PeriodicWork;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 
 import java.util.logging.Logger;
 
Index: jenkins/core/src/main/java/hudson/model/ExternalRun.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/model/ExternalRun.java	2012-02-10 15:39:22.693867337 +0000
+++ jenkins/core/src/main/java/hudson/model/ExternalRun.java	2012-02-28 16:49:18.505824788 +0000
@@ -26,7 +26,7 @@
 import hudson.Proc;
 import hudson.util.DecodingStream;
 import hudson.util.DualOutputStream;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 
 import javax.xml.stream.XMLInputFactory;
 import javax.xml.stream.XMLStreamException;
Index: jenkins/core/src/main/java/hudson/node_monitors/DiskSpaceMonitorDescriptor.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/node_monitors/DiskSpaceMonitorDescriptor.java	2012-02-10 15:39:22.709867337 +0000
+++ jenkins/core/src/main/java/hudson/node_monitors/DiskSpaceMonitorDescriptor.java	2012-02-28 16:49:18.505824788 +0000
@@ -29,7 +29,7 @@
 import hudson.Util;
 import hudson.slaves.OfflineCause;
 import hudson.node_monitors.DiskSpaceMonitorDescriptor.DiskSpace;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 
 import java.io.File;
 import java.io.IOException;
Index: jenkins/core/src/main/java/hudson/node_monitors/TemporarySpaceMonitor.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/node_monitors/TemporarySpaceMonitor.java	2012-02-10 15:39:22.709867337 +0000
+++ jenkins/core/src/main/java/hudson/node_monitors/TemporarySpaceMonitor.java	2012-02-28 16:49:18.505824788 +0000
@@ -31,7 +31,7 @@
 import jenkins.model.Jenkins;
 import hudson.node_monitors.DiskSpaceMonitorDescriptor.DiskSpace;
 import hudson.remoting.VirtualChannel;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 import org.kohsuke.stapler.DataBoundConstructor;
 
 import java.io.File;
Index: jenkins/core/src/main/java/hudson/security/AbstractPasswordBasedSecurityRealm.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/security/AbstractPasswordBasedSecurityRealm.java	2012-02-10 15:39:22.713867337 +0000
+++ jenkins/core/src/main/java/hudson/security/AbstractPasswordBasedSecurityRealm.java	2012-02-28 16:49:18.509824800 +0000
@@ -16,7 +16,7 @@
 import org.acegisecurity.userdetails.UserDetails;
 import org.acegisecurity.userdetails.UserDetailsService;
 import org.acegisecurity.userdetails.UsernameNotFoundException;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 import org.kohsuke.args4j.Option;
 import org.springframework.dao.DataAccessException;
 import org.springframework.web.context.WebApplicationContext;
Index: jenkins/core/src/main/java/hudson/tools/ZipExtractionInstaller.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/tools/ZipExtractionInstaller.java	2012-02-10 15:39:22.733867337 +0000
+++ jenkins/core/src/main/java/hudson/tools/ZipExtractionInstaller.java	2012-02-28 16:49:18.509824800 +0000
@@ -45,7 +45,7 @@
 import java.net.URLConnection;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.QueryParameter;
-import org.jvnet.animal_sniffer.IgnoreJRERequirement;
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
 
 /**
  * Installs a tool into the Hudson working area by downloading and unpacking a ZIP file.
